File: imc/imc-furtadinho.py
i=input
m=int(i())/float(i()**2)
s,p,o="Situacao:","Peso","Obeso"
print "IMC:","%.2f"%m,"\n"+s,["","Abaixo do "+p][m<20]+["",p+" Normal"][20<m<25]+["","Acima do "+p][25<m<30]+["",o][30<m<34]+["","Muito "+o][34<m]

File: imc/imc_furtadinho.rb
puts"IMC: %.2f"%m=gets.to_i/(gets.to_f**2).round(2),"Situacao: "+(m<20?"Abaixo do Peso":m<25?"Peso Normal":m<30?"Acima do Peso":m<34?"Obeso":"Muito Obeso")
